lib/plugins/plugin/lib/pluginService.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
```

Comment:
Is `pluginService` meant to be a place for common functionality? If so we should rename it to `utils.js` or something similar so that it's obvious when someone looks through the code.

Reply:
>Is pluginService meant to be a place for common functionality? 
Yup, will fix it.